helm/templates/query-service-config.yaml

```diff
@@ -35,6 +35,14 @@ data:
           connectionRetryBackoff = "{{ .Values.queryServiceConfig.data.postgresConnectionRetryBackoff }}"
         }
         {{- end }}
+        {{- if eq .Values.queryServiceConfig.clientType "trino" }}
```

Comment:
`.Values.queryServiceConfig.clientType` can have only one value. So this won't allow both Pinot + Trino together, (which will be needed to power two separate Explorer pages)?